src/utils/Utils.ts

```diff
@@ -1352,15 +1352,15 @@ export default class Utils {
   }
 
   public static generateKey(): string {
-    // TODO change 16 to 32 and test on Mac
+    // TODO: change 16 to 32 and test on Mac
     return crypto.randomBytes(16).toString('hex');
   }
 
   public static getDefaultKeyProperties(): CryptoKeyProperties {
     return {
       blockCypher: 'aes',
       blockSize: 256,
-      operationMode: 'ctr'
+      operationMode: 'gcm'
```

Comment:
This option is not working:
- Create a tenant
- Set up a connection in the Asset (where there is sensitive data) 
- Check the connection -> it fails

I replaced with 'ctr', set the password again in the Asset setting and it works.

Could you double check?